beater/v2_handler_test.go

```diff
@@ -0,0 +1,328 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
`errors` and `metrics` are missing completely from these tests. There is an open Issue for handling errors, but none for metrics. Please add an Issue or some tests.

Reply:
missed this one. We have an issue for adding tests for all the different event types here: https://github.com/elastic/apm-server/issues/1288